velox/dwio/dwrf/reader/ColumnReader.cpp

```diff
@@ -739,6 +739,116 @@ void TimestampColumnReader::next(
       valuesPtr, nullsPtr, secondsData, nanosData, numValues);
 }
 
+template <typename DataT>
+class DecimalColumnReader : public ColumnReader {
```

Comment:
Put this in anonymous class

Reply:
OK, has been moved to anonymous namespace